docs/stable/1-client.md

```diff
@@ -94,33 +96,39 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
+On top of this we run RLPx-based subprotocols, the client
+MAY use [Whisper v6](https://eips.ethereum.org/EIPS/eip-627), though the client 
+SHOULD use [Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md)
+for privacy-preserving messaging.
 
-There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+There MUST be a node that is capable of discovering peers and implements
+[Whisper v6](https://eips.ethereum.org/EIPS/eip-627),
+[Waku V0](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-0.md) and
+[Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md) specifications.
 
 #### Node discovery and roles
 
-There are four types of node roles:
-1. Bootstrap nodes
-2. Whisper relayers
-3. Mailservers (servers and clients)
-4. Mobile nodes (Status Clients)
+There are five types of node roles:
+1. `Bootstrap node`
+1. `Waku relayer`
```

Comment:
Probably shouldn't create a new role for Waku relayer. Just call it Waku or Whisper Relayer (or just Relayer?).
As In theory then you also have to split Mailserver and Mobile node, they both still us the Waku / Whisper protocol

Reply:
In this section I was implementing a suggestion from @oskarth https://github.com/status-im/specs/pull/114#discussion_r421938443 .

While rewriting this section I had the idea to simply rename the `Whisper relayer` to `Message relayer` and add a definition of a`Message relayer`. I decided to explicitly keep both `Whisper relayer` and `Waku relayer` so that deprecation would be simpler.

Yes I may eventually split the `Mailserver`, but I haven't rewritten that section yet and I wanted to see how divergent Whisper vs Waku Mailservers were.

Could you explain why the `Mobile node` would need to be split?

Reply:
I wouldn't split them, as the role is essentially the same, just a different protocol. As is the case for Mail server and mobile node. Yeah, perhaps Message relayer (Waku or Whisper) is better. Or just Waku/Whisper relayer.

Reply:
I like `Waku/Whipser relayer`, let's got with that.